src/frontend/js/utils/test/factories/joanie.ts

```diff
@@ -105,7 +105,6 @@ export const CertificateProductFactory = factory((): Product => {
     price_currency: faker.finance.currencyCode(),
     call_to_action: faker.lorem.words(3),
     certificate_definition: CertificationDefinitionFactory().one(),
-    orders: [],
```

Comment:
Why removing order ? 
I will use this in the enrollement certificate purchase/download dev in order to switch between purchase and download. (order.length > 0 ? download : purchase

Reply:
Because we do not use it anymore. Instead we try to fetch orders with validated or pending states. You will have to do the same about enrollments.